### PR TITLE
[Exporter.Geneva] Release 1.4.0-beta.5

### PR DESCRIPTION
**Pull request by abatishchev**
Follow up to #780.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #

### CODE REVIEW

**Review by Kielek — APPROVED** [review]

**Comment by abatishchev** [comment]
@Kielek, thanks! guide me please what's next?

File: src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+## 1.4.0-beta.5
```
Comment (Kielek) [inline src/OpenTelemetry.Exporter.Geneva/CHANGELOG.md:5]:
```suggestion
## 1.4.0-beta.5

Released 2022-Nov-21
```

**Comment by Kielek** [comment]
Someone from the Geneva component owners should agree on release https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/2658fd07fabbaadc14368e6bcf7fa5affebcf67a/.github/component_owners.yml#L11-L17 then @open-telemetry/dotnet-contrib-maintainers should make a release.

Fortunately, few developers are in both groups, so you have to wait more time. They are usually available in PST working hours.

**Comment by abatishchev** [comment]
Thanks for the correcting commit!

Since Geneva is pretty much internal to Microsoft and I'm a Microsoft employee as well, can you share a good point of contact or mention them in the issue to raise awareness?
Thanks again! Alex.

**Review by Yun-Ting — APPROVED** [review]

**Review by utpilla — APPROVED** [review]

**Review by reyang — APPROVED** [review]

**Comment by utpilla** [comment]
The package is published: https://www.nuget.org/packages/OpenTelemetry.Exporter.Geneva/1.4.0-beta.5

**Comment by abatishchev** [comment]
@utpilla thank you!

**Pull request merged**